packages/pipeline-editor/package.json

```diff
@@ -21,8 +21,11 @@
     "link": "yarn link"
   },
   "dependencies": {
-    "@elyra/canvas": "11.0.0",
+    "@elyra/canvas": "~12.5.0",
```

Comment:
why lock to an older minor version? I remember you explaining it somewhere but can't find the comment

Reply:
There's a bug that was introduced in 12.6 that throws a console error when JL triggers a drag event. I contacted the canvas people about it but I figured for now we could just pin to the minor.

Reply:
I'd say we wait till they fix that to merge unless it will delay our release then

Reply:
I think it's worth merging without those changes because we don't really know how high of a priority this is for them and this was a high priority for the upcoming release. One note though - it doesn't actually break anything in our code, it just shows a console error

Reply:
If it's just an upstream console error with no problems in elyra then I say let it in, it will go away automatically when we catch their release containing the fix